### PR TITLE
build(package): Update webpack-bundle-analyzer

### PR DESCRIPTION
**Pull request by mudlabs**
Update `webpack-bundle-analyzer` dependency to `3.3.2` to address vulnerability [WS-2019-0058](https://github.com/webpack-contrib/webpack-bundle-analyzer/issues/263)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -96,7 +96,7 @@
     "terser-webpack-plugin": "1.2.3",
     "ts-loader": "^5.3.1",
     "webpack": "~4.27.0",
-    "webpack-bundle-analyzer": "~3.0.2",
+    "webpack-bundle-analyzer": "^3.3.2",
```
Comment (KristianDD) [inline package.json]:
```suggestion
    "webpack-bundle-analyzer": "~3.3.2",
```

**Comment by rosen-vladimirov** [comment]
@mudlabs can you please handle @KristianDD 's comment. After that we can merge this PR

**Comment by mudlabs** [comment]
@rosen-vladimirov are we all good to merge this now?

**Comment by DimitarTachev** [comment]
test package_version#rc

**Pull request merged**